libraries/stdlib/src/kotlin/io/Console.kt

```diff
@@ -132,50 +136,89 @@ public inline fun println() {
     System.out.println()
 }
 
-// Since System.in can change its value on the course of program running,
-// we should always delegate to current value and cannot just pass it to InputStreamReader constructor.
-// We could use "by" implementation, but we can only use "by" with interfaces and InputStream is abstract class.
-private val stdin: BufferedReader by lazy { BufferedReader(InputStreamReader(object : InputStream() {
-    public override fun read(): Int {
-        return System.`in`.read()
-    }
+private const val BUFFER_SIZE: Int = 32
 
-    public override fun reset() {
-        System.`in`.reset()
-    }
+// Since System.in can change its value on the course of program running, we should always delegate to current value.
+private val stdin: InputStream
```

Comment:
This property seems no longer required, `System.in` can be used directly instead.